app/scripts/modules/core/src/application/application.less

```diff
@@ -164,3 +164,22 @@ input[type="checkbox"], input[type="radio"] {
   cursor: pointer;
 }
 
+@table-name: application;
+.generate-table(@table-name);
+
+.@{table-name}-table {
+  .table-header, .@{table-name}-row {
+    .@{table-name}-application {
+      left: 0;
```

Comment:
I don't think the `left` is doing anything here or any of the other columns

Reply:
you're right.  good catch.